<launch>

  <!-- Server machine setup -->
  <machine name="Sciame-Server" address="Sciame-Server" user="attilio"
	ros-root="$(env ROS_ROOT)"
	ros-package-path="$(env ROS_PACKAGE_PATH)"
	default="false" />

  <!-- Client 1 machine setup -->
  <machine name="fcd-04-ubuntu" address="fcd-04-ubuntu" user="erupter"
	ros-root="$(env ROS_ROOT)"
	ros-package-path="$(env ROS_PACKAGE_PATH)"
	default="false" />

  <!-- Saetta Base Node-->
  <node pkg="Saetta_Base" type="base" name="saetta11"/> </node>

  <!-- Saetta Joystick Teleop-->
  <node pkg="Saetta_Base" type"teleop_saetta" name="teleop_saetta11" machine="fcd-04-ubuntu"> </node>

</launch>
